incubator/magic-namespace/templates/rolebindings.yaml

```diff
@@ -0,0 +1,20 @@
+
+{{- $ctx := . }}
```

Comment:
IIRC, you can use `$` within a `range` to point to the outer context, no need to create a variable.